akka-stream-tests/src/test/scala/akka/stream/impl/GraphStageLogicSpec.scala

```diff
@@ -67,6 +68,18 @@ class GraphStageLogicSpec extends AkkaSpec with GraphInterpreterSpecKit with Con
     }
   }
 
+  object emitEmptyIterable extends GraphStage[SourceShape[Int]] {
+    val out = Outlet[Int]("out")
```

Comment:
for discussion in the future – we have 2 different styles now, sometimes "out" and sometimes "StageName-out", we should settle on one style, (not for this issue ofc)